tests/unit/linking/test_link.py

```diff
@@ -139,11 +141,12 @@ def test_compare_identifier_match(self):
             evaluators=[
                 {"feature": "IDENTIFIER", "func": "func:recordlinker.linking.matchers.compare_probabilistic_fuzzy_match"},
             ],
-            rule="func:recordlinker.linking.matchers.rule_probabilistic_match",
-            kwargs={"log_odds": {"IDENTIFIER": 0.35}, "true_match_threshold": 0.3},
+            rule="func:recordlinker.linking.matchers.rule_probabilistic_sum",
+            possible_match_window=(0.8, 0.925),
+            kwargs={"log_odds": {"IDENTIFIER": 0.35}},
         )
 
-        assert link.compare(rec, pat, algorithm_pass) is True
+        assert link.compare(rec, pat, algorithm_pass) == 0.35
```

Comment:
```suggestion
        assert link.compare(rec, pat, algorithm_pass) == algorithm_pass.kwargs["log_odds"]["IDENTIFIER"]
```

Reply:
nit: this makes it clearer that the resulting score is expecting to be equal to the log-odds and not normalized to 1.0. Might consider for other tests as well if you think it's a good idea.

Reply:
Yeah that's fair. I updated the other test that uses `identifier` to follow suit, since it's ambiguous. The other tests all have numbers above 5 or 6, so I don't think normalization is as much a concern, and to write out their decimal would be calling `round( jaro-winkler-similarity( name1, name2 ) * log-odds[FIRST_NAME] ) etc` and at that point it feels redundant to me since that's just reproducing the function code.